apiserver/facades/client/subnets/subnets_test.go

```diff
@@ -36,31 +38,50 @@ type SubnetTestMockSuite struct {
 	api *subnets.API
 }
 
-var _ = gc.Suite(&SubnetTestMockSuite{})
+var _ = gc.Suite(&SubnetSuite{})
 
-func (s *SubnetTestMockSuite) TearDownTest(c *gc.C) {
+func (s *SubnetSuite) TearDownTest(c *gc.C) {
 	s.api = nil
 }
 
-func (s *SubnetTestMockSuite) TestAllSpaces(c *gc.C) {
```

Comment:
Even though we are dropping AllSpaces, should the test not stay?

Reply:
The test is still in the old stub-based suite. This was a previously added canary for the mocks suite.